R/produce_at_sds_episodes_tests.R

```diff
@@ -0,0 +1,39 @@
+#' Alarm Telecare and SDS Episodes Tests
+#'
+#' @description Produce the test for the Alarm Telecare and SDS all episodes
+#'
+#' @param data new or old data for testing summary flags
+#' (data is from [get_sc_ch_episodes_path()])
+#'
+#' @return a dataframe with a count of each flag.
+#'
+#' @export
+#' @family social care test functions
+produce_at_sds_episodes_tests <- function(data) {
```

Comment:
```suggestion
produce_sc_at_tests <- function(data) {
```
Can we rename to this please? I was getting confused with having SDS in the name for this function as we should split these and make them specific to the tests